src/sentry/lang/javascript/processor.py

```diff
@@ -220,9 +220,12 @@ def fetch_file(url, project=None, release=None, allow_scraping=True):
 
     Attempts to fetch from the cache.
     """
-    cache_key = 'source:cache:v2:%s' % (
-        md5(url).hexdigest(),
-    )
+    if url is None:
```

Comment:
In theory, it's blocking it down below in L606. But this just makes `fetch_file` itself a bit safer since it's used in a few different places (and would likely be used more in the future). Seems better than guaranteeing that `fetch_file` _never_ is given `None`.